include/openssl/crypto.h

```diff
@@ -86,33 +86,28 @@ OPENSSL_EXPORT int CRYPTO_needs_hwcap2_workaround(void);
 
 // Data-Independent Timing (DIT) on AArch64
 
-#if defined(OPENSSL_AARCH64) && !defined(OPENSSL_WINDOWS) && defined(MAKE_DIT_AVAILABLE)
+#if defined(OPENSSL_AARCH64) && !defined(OPENSSL_WINDOWS)
```

Comment:
* Instead of "disallowing" Windows, we should just explicitly "allow" Linux and Apple. There are other OS's, like [FreeBSD](https://www.freebsd.org/platforms/arm/), for which the `ARMV8_DIT_ALLOWED` macro can not be used properly.(?)
* Perhaps we need a meta-macro that we can use in the places that we guard the DIT functionality?
```
#define AARCH64_DIT_SUPPORTED (defined(OPENSSL_AARCH64) && (defined(OPENSSL_LINUX) || defined(OPENSSL_APPLE))
```